src/components/character-count/threshold/index.njk

```diff
@@ -5,12 +5,22 @@ layout: layout-example.njk
 
 {% from "govuk/components/character-count/macro.njk" import govukCharacterCount %}
 
+{% set textareaValue -%}
+Type more letters into this field after this message to see the threshold feature.
+
+GOV.UK includes services, information and guidance that only government can provide. It’s designed so that users do not need to understand the complex structures of government in order to complete their task. Its content and design is user-centred so that interacting with government is accessible to everybody.
+
+GOV.UK is the official source of government information. It is impartial so that people trust the content they find.
+
+Content and services are continually improved to meet user needs. You can visit the GOV.UK roadmap for the most up to date plan for future improvements.
+{%- endset %}
+
 {{ govukCharacterCount({
   name: "threshold",
   id: "threshold",
-  maxlength: 112,
-  threshold: 75,
-  value: "Type another letter into this field after this message to see the threshold feature",
+  maxlength: 840,
+  threshold: 80,
```

Comment:
```suggestion
  maxlength: 1000,
  threshold: 260,
```

Reply:
I know it's a cumbersome ask because it would require more content, but is there any way we can get the threshold to be more like a 75%+ threshold, to be more realistic? I can't think of a use-case where we would want to activate the threshold condition at something as low as 26%.

Reply:
I added a suggested update tot he content which should bring it to around 849 characters. Then we can go for an 85% threshold on a 100 character limit.

Reply:
To go with ~850 character `textareaValue` value suggested aboive

```suggestion
  maxlength: 1000,
  threshold: 85,
```